server/issue.go

```diff
@@ -398,7 +398,7 @@ func (p *Plugin) GetCreateIssueMetadataForProjects(instanceID, mattermostUserID
 		return nil, err
 	}
 
-	return client.GetCreateMetaInfo(&jira.GetQueryOptions{
+	return client.GetCreateMetaInfo(p.API, &jira.GetQueryOptions{
```

Comment:
You can also pass `p.client` here and see if you can use that. See what is used mostly to do API calls in this plugin - `p.API` or `p.client`.

Reply:
actually in one of the api in the same interface they passed p.API to log so I have used the same